src/jobs/activities/transfer-activity.ts

```diff
@@ -53,6 +53,7 @@ export type TransferEventData = {
   fromAddress: string;
   toAddress: string;
   amount: number;
+  amountString?: string;
```

Comment:
Why `AmountString` here?

Reply:
This was added just to make sure that the raw value of amount(BigNumber/BigInt) is sent as is. And the amountString is sent separately because I was running into some issues related to version of ethers (v5/v6).

But, it can be removed as the handling on pacman side works fine with just amount.